README.md

```diff
@@ -183,9 +178,11 @@ class Program
                 catch (ConsumeException e)
                 {
                     Console.WriteLine($"Error occured: {e.Error.Reason}");
+                    break;
```

Comment:
I thought we dont want to break out on errors.

Reply:
forgot about that. the reason I keep trying to put this in is the `Close` at the end is pointless as there is no exit condition on the loop. but we want this in the example so people know to use it. we also don't want to exit the loop as you say (now you point this out, again). i'll take out the break, and have the pointless `Close` rather than make the example longer.

Reply:
Ctrl-C is a good way to show-case termination.